src/components/TopNavigation/WishlistTopNavigation.jsx

```diff
@@ -0,0 +1,44 @@
+// All module imports
+import { Link } from "react-router-dom";
+
+// All local imports
+import "./TopNavigation.css";
+import logo from "../../assets/website-logo.png";
```

Comment:
add all kinds of assets in the public folder , and then access it (good and a must practice), search for how to access public folder files in react if need help

Reply:
Will do it later after reading more! Thanks, saved this suggestion